blocks_to_transfers/set_pickup_drop_off.py

```diff
@@ -0,0 +1,17 @@
+from gtfs_loader.schema import TransferType, PickupType
+
+def set_pickup_drop_off(gtfs):
+    from_trip_ids_to_set = set()
+    to_trip_ids_to_set = set()
+    for from_trip_id, transfers in gtfs.transfers.items():
+        for transfer in transfers:
+            transfer_type = transfer.transfer_type
+            to_trip_id = transfer.to_trip_id
+
+            if not (from_trip_id and to_trip_id):
+                continue
```

Comment:
```suggestion
            if not (from_trip_id and to_trip_id):
                continue
                
```

I'm not a stickler for it but I usually leave a blank line between different if blocks